### ISSUE TITLE
Quantile too tight for boosting mixers

### ISSUE THREAD

**Issue by George3d6** [mindsdb/lightwood#200]
Boosting mixers overfit their quantiles and thus they end up being "too tight" => very low accuracy in our new accuracy determination method for numerical values.

We should fix this by using a validation dataset and trying different quantile values for the helper mixers until we get the *right* accuracy on the validation set or at least the closest possible one within some boundaries.

Here the "right" accuracy meaning that which is closest to the one that would be predicted by the actual quantiles.

**Comment by George3d6** [comment]
quantiles are removed now
